lib/voice.js

```diff
@@ -30,7 +30,7 @@ Voice.prototype.call = function (params) {
                         }
                     };
                 }
-                if (!(/^\+\d{1,3}\d{3,}$/).test(value)) {
+                if (!(/^(\+\d{1,3}\d{3,})(,\+\d{1,3}\d{3,})*$/).test(value)) {
```

Comment:
Why are you duplicating the phone_number regex?

Reply:
It was quickest regex I could write to accept at least 1 phone number, i.e. `+25412345678` and `+25412345678,+25412345678`. There's probably a simpler way to write it, lemme check

Reply:
@kevokevoh can't find a simpler/shorter way to write that regex :grimacing:

Reply:
@aksalj that should work, will update this ASAP